src/patches.rs

```diff
@@ -249,7 +249,7 @@ fn add_skip_hudmemos_strgs(pickup_resources: &mut HashMap<(u32, FourCC), structs
     }
 }
 
-fn build_artifact_temple_totem_scan_strings<R>(pickup_layout: &[u8], rng: &mut R)
+fn build_artifact_temple_totem_scan_strings<R>(pickup_layout: &[u8], rng: &mut R, showHints: bool)
```

Comment:
`showHints` should be `show_hints`.

Reply:
Sorry about that, I autopilot with camelCase as opposed to snake_case. The PR should have the snake_case fix now.